pkg/subctl/cmd/deploybroker.go

```diff
@@ -73,6 +74,8 @@ func init() {
 
 	deployBroker.PersistentFlags().BoolVar(&operatorDebug, "operator-debug", false, "enable operator debugging (verbose logging)")
 
+	deployBroker.PersistentFlags().StringVar(&brokerNamespace, "broker-namespace", broker.SubmarinerBrokerNamespace, "namespace for broker")
```

Comment:
I think we should move `broker.SubmarinerBrokerNamespace` here, and make it a tool-specific default. We can pretty much get rid of `pkg/broker/namespace.go` entirely given that `NewBrokerNamespace()` is trivial (it can be merged in `CreateNewBrokerNamespace()`).